app/helpers/application_helper.rb

```diff
@@ -550,4 +550,16 @@ def first_paragraph(full_text, placeholder_text = 'No preview available.')
       return content_tag(:p, ts(placeholder_text), class: 'placeholder')
     end
   end
+
+  # change the default link renderer for will_paginate
+  def will_paginate(collection_or_options = nil, options = {})
```

Comment:
Does this mean we can remove the `will_paginate` method in config/initializers/gem-plugin_config/will_paginate_config.rb?